0021-plugin-rpc-api/0021-plugin-rpc-api.md

```diff
@@ -1,21 +1,42 @@
 ---
 title: Plugin RPC API
-draft: 2
+draft: 3
 ---
 # Plugin RPC API
 
+This spec depends on the [LPI spec](../0004-ledger-plugin-interface/).
+
 ## Description
 
-> TODO
+This specification details how to serialize Interledger's Ledger Plugin Interface over http.
 
 ## Specification
 
-### Send Transfer
+For every method of the LedgerPlugin class, it defines a Remote Procedure Call (RPC) method.
+All RPC methods are implemented as a http or
+https POST to an endpoint URL, with the method name as the 'method' query parameter.
+The post body is a JSON document representing the array of method arguments.
+
+For each `incoming_*` event defined by the LPI spec, a corresponding RPC call will be made in the opposite direction,
+namely:
+* `incoming_transfer` -> `sendTransfer`
+* `incoming_fulfill` -> `fulfillCondition`
+* `incoming_reject` -> `rejectIncomingTransfer`
+* `incoming_request` -> `sendRequest`
+
+Note incoming requests are a special case, because they require a response.
+Instead of taking the request response from the currently registered request handler, a http call for the `sendRequest`
+method is made, and its response body parsed as a JSON document.
+
+Calls to `connect` and `disconnect`, `registerRequestHandler` and `deregisterRequestHandler`
```

Comment:
Huh? I don't think we need to define anything for that method in this protocol. That's a method that makes sense locally but has no remote meaning